src/lib/datepicker/multi-year-view.html

```diff
@@ -3,7 +3,6 @@
     <tr><th class="mat-calendar-table-header-divider" colspan="4"></th></tr>
   </thead>
   <tbody mat-calendar-body
-         allowDisabledSelection="true"
```

Comment:
@mmalerba this behavior seems pretty deliberate, but I'm not sure whether it makes sense. Also it only applies to mouse users, because you can't reach disabled cells with the keyboard.

Reply:
It looks like its been that way since the beginning, but I don't remember why I did it, and I agree that it doesn't really make sense. Can we also remove the `allowDisabledSelection` input from calendar body?

Reply:
Done.